pyuvdata/tests/test_ms.py

```diff
@@ -135,11 +150,29 @@ def test_readUVFITS_readMS():
         DATA_PATH, 'day2_TDEM0003_10s_norx_1src_1spw.uvfits')
     uvtest.checkWarnings(uvfits_uv.read_uvfits, [uvfits_file],
                          message='Telescope EVLA is not')
-    uvtest.checkWarnings(ms_uv.read_ms, [ms_file, True, True, 'DATA', 'AIPS', True],
+    uvtest.checkWarnings(ms_uv.read_ms, [ms_file, True, True, 'DATA', 'AIPS'],
```

Comment:
This doesn't look like it should be working. The positional arguments for read_ms are:
`filepath, run_check, check_extra, run_check_acceptability, data_column, and pol_order`
So three booleans, but only two are being passed.
Also, the extra arguments are actually not needed if they're being set to the default anyway. It looks like the original line had all these to get to the last keyword.
But....
checkWarnings also takes an argument `func_kwargs`, which is a dictionary to pass keywords to the function in question (read_ms here). So the old line would have looked like:
`uv.test.checkWarnings(ms_uv.read_ms, [ms_file], {test_import_uvfits=True}, ...`
Just an FYI for future use of checkWarnings.